Source/ACE.Server/WorldObjects/Gem.cs

```diff
@@ -101,6 +103,87 @@ public override void ActOnUse(WorldObject activator)
                     player.Session.Network.EnqueueSend(new GameMessageSystemChat($"{Name} accepted. Click on the quill icon in the lower right corner to open your contract tab to view your active contracts.", ChatMessageType.Broadcast));
             }
 
+            if (UseCreateItem.HasValue && UseCreateItem > 0)
```

Comment:
many of these UseCreateItem.HasValues are redundant

this is equivalent to if (UseCreateItem > 0)